ev3dev2/motor.py

```diff
@@ -89,11 +95,20 @@ def __init__(self, percent):
     def __str__(self):
         return str(self.percent) + "%"
 
+    def __mul__(self, other):
+
+        if isinstance(other, SpeedPercent):
+            return SpeedPercent(self.percent * other.percent)
+        elif isinstance(other, (float, int)):
+            return SpeedPercent(self.percent * other)
+        else:
+            raise Exception("{} can only be multiplied by a SpeedPercent, int, or float".format(self))
+
     def to_native_units(self, motor):
         """
         Return this SpeedPercent in native motor units
         """
-        return self.percent / 100 * motor.max_speed
+        return round((self.percent * motor.max_speed) / 100)
```

Comment:
speed_sp must be an integer so I changed all of the to_native_unit() calls to round() what they are returning

Reply:
See related discussion here: https://github.com/ev3dev/ev3dev-lang-python/pull/500#discussion_r214096109

Reply:
Also, it looks like failing tests on Travis may be caused by this change.

Reply:
ah I missed those round() calls...mine are not needed will chop them.